docs/dns.md

```diff
@@ -1,12 +1,10 @@
 # DNS Support
-If you point a domain or subdomain to Google Cloud DNS, you can use `kctf-setup-dns` and `kctf-chal-dns` to map *[challenge_name].yourdomain.com* to your challenge.
+If you point a domain or subdomain to Google Cloud DNS as an NS record, kCTF will create a DNS zone that maps the challenge to your challenge's IP address.
 
 Follow these steps:
 
 1. Configure the subdomain in `kctf-setup-config-create`. 
 
-   Example: If you enter *foo.example.com*, the challenges will be deployed on *[challenge-name].foo.example.com*.
-1. Run `kctf-setup-dns` to create the DNS zone in Google Cloud DNS.
+   Example: If you enter *foo.example.com*, the challenges will be deployed on *xxxxx.foo.example.com*.
```

Comment:
cosmetic nit, you can simplify by removing the future tense without changing the meaning: "...the challenges are deployed on..."